actor/src/main/resources/reference.conf

```diff
@@ -741,6 +741,9 @@ pekko {
     # to reduce noise.
     warn-on-no-serialization-verification = on
 
+    # list of fqcn of classes that may use non-pekko serializer within pekko package without warn log
+    warn-non-pekko-serializer-allow-list = ${?pekko.actor.warn-non-pekko-serializer-allow-list} []
```

Comment:
might be worth updating the comment to mention that this is new to Pekko v1.1.0

Reply:
added

Reply:
What's the usercase for this :) I'm asking because I'm not a persistence user:)